en_US.json

```diff
@@ -1602,6 +1603,7 @@
 	"property.origin.saveButton": "Save",
 	"property.origin.server": "Servers *",
 	"property.origin.server.info": "Please enter at least one valid server. If you do not specify HTTP and HTTPS ports, the respective defaults 80 and 443 will be used.",
+	"property.origin.server.url.ipv6use.alertMessage": "You've used an IPv6 address origin server. You must enable IPv6 origin access under Advanced Settings.",
```

Comment:
I suggest writing "You have" instead of the contraction "You've"

"You have specified an IPv6 address for an origin server."

Reply:
Followed the existing alert message style. Will update to use "You have" instead of "You've"

Reply:
Updated as per suggestion.